app/mattermost.js

```diff
@@ -229,20 +200,20 @@ export default class Mattermost {
         return true;
     };
 
-    handleConfigChanged = async (serverVersion) => {
+    handleServerConfigChanged = async (serverVersion) => {
         const {dispatch, getState} = this.store;
         const version = serverVersion.match(/^[0-9]*.[0-9]*.[0-9]*(-[a-zA-Z0-9.-]*)?/g)[0];
         const intl = this.getIntl();
         const state = getState();
 
         if (serverVersion) {
-            if (semver.valid(version) && semver.lt(version, Config.MinServerVersion)) {
+            if (semver.valid(version) && semver.lt(version, LocalConfig.MinServerVersion)) {
                 Alert.alert(
                     intl.formatMessage({id: 'mobile.server_upgrade.title', defaultMessage: 'Server upgrade required'}),
                     intl.formatMessage({id: 'mobile.server_upgrade.description', defaultMessage: '\nA server upgrade is required to use the Mattermost app. Please ask your System Administrator for details.\n'}),
                     [{
                         text: intl.formatMessage({id: 'mobile.server_upgrade.button', defaultMessage: 'OK'}),
-                        onPress: this.handleVersionUpgrade
+                        onPress: this.handleAppVersionUpgraded
```

Comment:
Is this an app version upgrade or a server version upgrade? The help text sounds like a server one